tap_google_ads/__init__.py

```diff
@@ -12,14 +12,11 @@
 
 REQUIRED_CONFIG_KEYS = [
     "start_date",
-    "oauth_client_id",
-    "oauth_client_secret",
-    "refresh_token",
-    "customer_ids",
```

Comment:
We will need to support backwards compatibility the developer token auth and service account auth methods.   And also removing this will make it difficult for this fork to get merged back into the upstream repo.